GamerpotFrontend/src/components/Carousel/styles.ts

```diff
@@ -0,0 +1,18 @@
+import {StyleSheet} from 'react-native';
+import {HEIGHT, WINDOW_WIDTH} from '../../library/constants/carouselConstants';
+
+const width = WINDOW_WIDTH;
+const height = HEIGHT;
```

Comment:
Why are you doing this assignment instead of using the constants you are importing?

Reply:
I forgot it when I was refactoring. Before I had a constant with the value right there, I think it was easier for me to assign it to the variable already declared. I'll remove it and make direct use of the imported constant.